docs/getting-started/tools/index.md

```diff
@@ -33,6 +33,8 @@ environment.
   scenarios. You can also use multiple browsers to have different browser extension version
   installed to compare them.
 - [Docker](https://docs.docker.com/get-docker/) - required for server development only
+- [Dotnet SDK](https://dotnet.microsoft.com/download) - v8.x required for server development on some
```

Comment:
```suggestion
- [.NET SDK](https://dotnet.microsoft.com/download) - required for server and other backend development
```

Reply:
Likely needs a split line.

Reply:
It did. This was auto-resolved when I took the suggestion.